src/docs/Foundations/Layout/Grid/index.mdx

```diff
@@ -1,23 +1,20 @@
 ---
 title: 'Grid system'
-order: 3
+order: 2
+description: 'A grid is used to create a **more structured and more harmonious design.** It’s the framework of your page, you build the layout by placing blocks of contents (text, images,…) on a certain number of columns.'
 status:
   sketch: 'stable'
   scss: 'stable'
 ---
 
-# Overview
+> A grid is used to create a **more structured and more harmonious design.** It’s the framework of your page, you build the layout by placing blocks of contents (text, images,…) on a certain number of columns. Gutters are used to separate 2 blocks, they give air to the design. Columns and gutters form a container. The container can be **full width or with a fixed size.**
```

Comment:
To avoid confusion with the developer's term, can we use **frame** instead of **framework** ?